src/app/system/roles-and-permissions/add-role/add-role.component.ts

```diff
@@ -0,0 +1,61 @@
+/** Angular Imports */
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router, ActivatedRoute } from '@angular/router';
+
+/** Custom Services */
+import { SystemService } from '../../system.service';
+
+/**
+ * Add Role Component.
+ */
+@Component({
+  selector: 'mifosx-add-role',
+  templateUrl: './add-role.component.html',
+  styleUrls: ['./add-role.component.scss']
+})
+export class AddRoleComponent implements OnInit {
+
+  /** Role form. */
+  roleForm: FormGroup;
+
+  /**
+   * @param {FormBuilder} formBuilder Form Builder.
+   * @param {SystemService} systemService Accounting Service.
```

Comment:
Service incorrectly mentioned as accounting here.

Reply:
After you commented about the Accounting Service, I went to check all the other components and many of them also has the comment as Accounting Service. I've changed the ones with this to System Service in the new push.

Reply:
Oh, good catch. But don't change them in this PR. This PR should only be for the addition of add role component.